eternalcore-core/src/main/java/com/eternalcode/core/configuration/migrations/Migration_0010_Move_jail_to_dedicated_section.java

```diff
@@ -0,0 +1,37 @@
+package com.eternalcode.core.configuration.migrations;
+
+import static eu.okaeri.configs.migrate.ConfigMigrationDsl.move;
+
+import eu.okaeri.configs.migrate.builtin.NamedMigration;
+
+public class Migration_0010_Move_jail_to_dedicated_section extends NamedMigration {
```

Comment:
rename jail section, you aren ot moving to dedicated section, you are just renaming.